src/components/organisms/connections/table.tsx

```diff
@@ -53,6 +52,7 @@ export const ConnectionsTable = () => {
 	};
 
 	const handleAction = (action: "edit" | "add", connectionId: string) => {
+		setConnectionInProgress(false);
```

Comment:
Remove it, please, and add:
```
 finally {
	setConnectionInProgress(false);
}
```

in useConnectionForm.ts in the end of the `createConnection` function.